index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Use lang to show the prefered lenguage. <html lang="en">

Reply:
kindly phrase the comments politely, as suggestions instead of orders. For example:
* This is a polite suggestion: *"I think it would be better to use semantic tags"*
* This reads as an order: *"Use semantic tags"*

Reply:
Hi @GuilhermeGiachellin, your comment is pretty polite 👍, however, you've not stated why the student should use semantic tags. Also, try to always double-check for any typos.